x-pack/test/api_integration/services/index.ts

```diff
@@ -42,4 +44,6 @@ export const services = {
   indexManagement: IndexManagementProvider,
   slo: SloApiProvider,
   securitySolutionApi: SecuritySolutionApiProvider,
+  alertingApi: AlertingApiProvider,
+  sloApi: SloApiProviderNew,
```

Comment:
I believe `sloApi: SloApiProviderNew,` is a copy of `SloApiProvider` that I already migrated to deployment-agnostic format.